Task2/src/app/weatherList/list.component.ts

```diff
@@ -0,0 +1,84 @@
+import { Component, Input, Output, OnChanges, EventEmitter } from '@angular/core';
```

Comment:
I don't really understand how list and pagination could be separated, because they can't be used one without another.
Please let's talk about it tomorrow.